templates/20years/index.html

```diff
@@ -708,31 +708,23 @@ <h2>
           <div class="p-heading-icon">
             <div class="p-heading-icon__header">
               <img class="p-heading-icon__img"
-                   src="https://assets.ubuntu.com/v1/5d57ae90-@jono%20bacon.png"
-                   alt="" />
+                   src="https://assets.ubuntu.com/v1/523f779c-Shannon%20smith.jpeg"
+                   alt="Shannon Smith photo" />
               <p class="p-heading-icon__title">
-                <strong><small>LinkedIn post by Jono Bacon</small></strong>
+                <strong><small>LinkedIn post by Shannon Smith</small></strong>
               </p>
             </div>
           </div>
-          <p>The story of a company that changed my life forever...</p>
-          <div class="p-section--shallow">
-            {{ image(url="https://assets.ubuntu.com/v1/081ef9b7-post%203.png",
-                        alt="",
-                        width="1512",
-                        height="851",
-                        hi_def=True,
-                        loading="lazy") | safe
-            }}
-          </div>
+          <p>20 years of Ubuntu , the OS that we all love ❤️</p>
+          <p>What's your favourite release?</p>
```

Comment:
This isn't in the original post https://www.linkedin.com/posts/shannonandriansmith_ubuntults-opensource-activity-7191994640412151808-KUQW?utm_source=share&utm_medium=member_desktop